cmd/nodecmd/wiz.go

```diff
@@ -253,23 +255,38 @@ func wiz(cmd *cobra.Command, args []string) error {
 	if err = deployClusterYAMLFile(clusterName, subnetName); err != nil {
 		return err
 	}
-	ux.Logger.GreenCheckmarkToUser("Cluster information YAML file can be found at /home/ubuntu/clusterInfo.yaml at external host")
 	return nil
 }
 
 func deployClusterYAMLFile(clusterName, subnetName string) error {
-	separateHosts, err := ansible.GetInventoryFromAnsibleInventoryFile(app.GetMonitoringInventoryDir(clusterName))
-	if err != nil {
-		return err
+	var separateHosts []*models.Host
+	var err error
+	if utils.FileExists(app.GetMonitoringInventoryDir(clusterName)) {
```

Comment:
nit: `app.GetMonitoringInventoryDir(clusterName)` - used multiple times, let's assign variable for this